docs/developers/talawa-api/mutations.md

```diff
@@ -3,10 +3,11 @@ id: mutations
 title: Mutations
 ---
 
+> Mutations can be classified in a number of ways, including by the type of change that they introduce, the part of the page that they affect, or the manner in which they are introduced. Talawa API introduces mutation to alter it's value to a certain degree.
+
 ## Mutations According to Roles
-This doucument outlines our mutations and to which type of user each is constrained
+This doucument outlines our mutations and to which type of user each is constrained. Common types of mutations include insertions and deletions among others. Insertions are new changes to the Talawa API, while deletions removes.
```

Comment:
Please fix the legacy error in the spelling of document.  Please remove the words "among others"

Reply:
@faveee  Please remove the words "among others"